src/code/z_view.c

```diff
@@ -69,15 +69,15 @@ void func_800AA358(View* view, Vec3f* eye, Vec3f* vec2, Vec3f* vec3) {
     }
 
     view->eye = *eye;
-    view->unk_34 = *vec2;
-    view->unk_40 = *vec3;
+    view->lookAt = *vec2;
+    view->up = *vec3;
```

Comment:
vec2 and vec3 arguments here could be named lookAt and up, as well as in the other function.

Reply:
done